From: "Dr. Tobias Quathamer" <toddy@debian.org>
Date: Fri, 27 Apr 2018 10:01:22 +0200
Subject: Better explanation how to read the documentation.

This patch is based on a suggestion from Ben Wong.

Thanks: Ben Wong <galadriel@wongs.net>
Closes: #874680
---
 trek/trek.6.in | 19 +++++++++++++++----
 1 file changed, 15 insertions(+), 4 deletions(-)

diff --git a/trek/trek.6.in b/trek/trek.6.in
index e3cc135..9d4b0a1 100644
--- a/trek/trek.6.in
+++ b/trek/trek.6.in
@@ -75,10 +75,6 @@ You should normally start out with a novice and work up.
 .Pp
 In general, throughout the game, if you forget what is appropriate
 the game will tell you what it expects if you just type in a question mark.
-.Sh AUTHOR
-.An Eric Allman
-.Sh SEE ALSO
-.Pa @docdir@/trek.me
 .Sh COMMAND SUMMARY
 .Bl -item -compact
 .It
@@ -137,3 +133,18 @@ amt1 course1 spread1 ...
 .It
 .Ic w Ns arp warp_factor
 .El
+.Sh AUTHOR
+.An Eric Allman
+.Sh SEE ALSO
+.Pp
+The complete documentation,
+.Em Trek
+by Eric Allman, is in
+.Pa @docdir@/trek.me.gz
+which can be viewed using
+.Dl cd @docdir@
+.Dl zcat trek.me.gz | tbl | nroff -me | less -r
+.Pp
+Or you can view a PDF version of the documentation,
+available here:
+.Dl @docdir@/trek.pdf
